### PR TITLE
Zoo search 1: updated style, index & about.

### PR DESCRIPTION
**Pull request by LuisChevere**
Updated versions of the Index.html, the about.html & the style css.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>about me</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/purecss@3.0.0/build/pure-min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/purecss@3.0.0/build/grids-responsive-min.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <script src=""></script>
```
Comment (anthonynguyen-dev) [inline about.html]: This is missing a script src.  This also should be at the bottom of the body.

Comment (briannabgb) [inline about.html]: missing script source and needs to be at the bottom of the body2e

File: index.html [hunk 2]
```diff
@@ -2,45 +2,40 @@
...
         </div>
-        <a href="https://www.rainn.org/articles/safety-tips-traveling">CDC Guide</a>
-    </footer>
+        <div class=" pure-u-sm-8-24 pure-u-md-8-24 pure-u-lg-8-24 pure-u-xl-8-24" id="bigLogo">LOGO</div>
+    </div>
 
 </body>
 
-<script src=""></script>
 <script src=""></script>
```
Comment (anthonynguyen-dev) [inline index.html:39]: This script is empty.

**Review by briannabgb — CHANGES_REQUESTED** [review]

File: about.html (hunk 1 above)
Comment (LuisChevere) [reply about.html]: I removed it.

File: index.html (hunk 2 above)
Comment (LuisChevere) [reply index.html:39]: This will be added later.

File: about.html (hunk 1 above)
Comment (LuisChevere) [reply about.html]: This will be added later.

**Review by anthonynguyen-dev — APPROVED** [review]
LGTM

**Review by briannabgb — APPROVED** [review]
LGTM

**Pull request merged**